projects/1.18/assets/create-cafe/createcafe/lang/zh_cn.json

```
@@ -1 +1,170 @@
+  "block.createcafe.lime_tea": "酸橙味奶茶",
+  "item.createcafe.strawberry_milk_tea": "草莓味奶茶",
+  "block.createcafe.strawberry_tea": "草莓味奶茶",
+  "item.createcafe.peach_milk_tea": "桃味奶茶",
+  "block.createcafe.peach_tea": "桃味奶茶",
+  "item.createcafe.plum_milk_tea": "李子味奶茶",
+  "block.createcafe.plum_tea": "李子味奶茶",
+  "item.createcafe.jackfruit_milk_tea": "菠萝蜜味奶茶",
+  "block.createcafe.jackfruit_tea": "菠萝蜜味奶茶",
+  "item.createcafe.apricot_milk_tea": "杏味奶茶",
+  "block.createcafe.apricot_tea": "杏味奶茶",
+  "item.createcafe.durian_milk_tea": "榴莲味奶茶",
+  "block.createcafe.durian_tea": "榴莲味奶茶",
+  "item.createcafe.fig_milk_tea": "无花果味奶茶",
+  "block.createcafe.fig_tea": "无花果味奶茶",
+  "item.createcafe.grape_milk_tea": "葡萄味奶茶",
+  "block.createcafe.grape_tea": "葡萄味奶茶",
+  "item.createcafe.grapefruit_milk_tea": "柚子味奶茶",
+  "block.createcafe.grapefruit_tea": "柚子味奶茶",
+  "item.createcafe.starfruit_milk_tea": "杨桃味奶茶",
+  "block.createcafe.starfruit_tea": "杨桃味奶茶",
+  "item.createcafe.blood_orange_milk_tea": "血橙味奶茶",
+  "block.createcafe.blood_tea": "血橙味奶茶",
+  "item.createcafe.lychee_milk_tea": "荔枝味奶茶",
+  "block.createcafe.lychee_tea": "荔枝味奶茶",
+  "item.createcafe.blueberry_milk_tea": "蓝莓味奶茶",
+  "block.createcafe.blueberry_tea": "蓝莓味奶茶",
+  "item.createcafe.pineapple_milk_tea": "菠萝味奶茶",
+  "block.createcafe.pineapple_tea": "菠萝味奶茶",
+  "item.createcafe.apple_milk_tea": "苹果味奶茶",
+  "block.createcafe.apple_tea": "苹果味奶茶",
+  "item.createcafe.pumpkin_milk_tea": "南瓜味奶茶",
+  "block.createcafe.pumpkin_tea": "南瓜味奶茶",
+  "item.createcafe.avocado_milk_tea": "鳄梨味奶茶",
+  "block.createcafe.avocado_tea": "鳄梨味奶茶",
+  "item.createcafe.sweetberry_milk_tea": "甜浆果味奶茶",
+  "block.createcafe.sweetberry_tea": "甜浆果味奶茶",
+  "item.createcafe.tamarind_milk_tea": "酸角味奶茶",
+  "block.createcafe.tamarind_tea": "酸角味奶茶",
+  "item.createcafe.papaya_milk_tea": "木瓜味奶茶",
+  "block.createcafe.papaya_tea": "木瓜味奶茶",
+  "item.createcafe.mana_berry_milk_tea": "魔力浆果味奶茶",
+  "block.createcafe.mana_tea": "魔力浆果味奶茶",
+  "item.createcafe.kiwi_milk_tea": "猕猴桃味奶茶",
+  "block.createcafe.kiwi_tea": "猕猴桃味奶茶",
+  "item.createcafe.orange_milk_tea": "橙味奶茶",
+  "block.createcafe.orange_tea": "橙味奶茶",
+  "item.createcafe.yucca_milk_tea": "丝兰味奶茶",
+  "block.createcafe.yucca_tea": "丝兰味奶茶",
+  "item.createcafe.aloe_milk_tea": "芦荟味奶茶",
+  "block.createcafe.aloe_tea": "芦荟味奶茶",
+  "item.createcafe.coconut_milk_tea": "椰子味奶茶",
+  "block.createcafe.coconut_tea": "椰子味奶茶",
+  "item.createcafe.persimmon_milk_tea": "柿子味奶茶",
+  "block.createcafe.persimmon_tea": "柿子味奶茶",
+  "item.createcafe.pomegranate_milk_tea": "石榴味奶茶",
+  "block.createcafe.pomegranate_tea": "石榴味奶茶",
+  "item.createcafe.raspberry_milk_tea": "山莓味奶茶",
+  "block.createcafe.raspberry_tea": "山莓味奶茶",
+  "item.createcafe.lemon_milk_tea": "柠檬味奶茶",
+  "block.createcafe.lemon_tea": "柠檬味奶茶",
+  "item.createcafe.watermelon_milk_tea": "西瓜味奶茶",
+  "block.createcafe.watermelon_tea": "西瓜味奶茶",
+  "item.createcafe.vanilla_milk_tea": "奶茶",
+  "block.createcafe.vanilla_tea": "奶茶",
```

```suggestion
  "item.createcafe.vanilla_milk_tea": "香草味奶茶",
  "block.createcafe.vanilla_tea": "香草味奶茶",
```
没看原文（）
不过应该是兼容Neapolitan的Vanilla（香草）

---

看了眼源码
支持`forge:crops/vanillabean`和`neapolitan:dried_vanilla_pods`
第一个应该是潘马斯的香草，第二个是那不勒斯风情的干香草荚
以第二个为参照（主要是由于里面有一大堆香草味的食物），一般都会把香草留着，这里这样应该好一些